### ISSUE TITLE
MKVToolNix fails for a mov videofile

### ISSUE THREAD

**Issue by Klaus1189** [mbunkus/mkvtoolnix#1630]
I can't open and export this file.

    Allgemein
    Vollständiger Name                       : file.mov
    Format                                   : MPEG-4
    Format-Profil                            : QuickTime
    Codec-ID                                 : qt   2005.03 (qt  )
    Dateigröße                               : 94,4 MiB
    Dauer                                    : 3min 49s
    Gesamte Bitrate                          : 3 446 Kbps
    Kodierungs-Datum                         : UTC 2010-01-04 04:45:20
    Tagging-Datum                            : UTC 2010-01-04 08:22:19
    verwendete Encoder-Bibliothek            : Apple QuickTime

    Video
    ID                                       : 1
    Format                                   : AVC
    Format/Info                              : Advanced Video Codec
    Format-Profil                            : Main@L3
    Format-Einstellungen für CABAC           : Nein
    Format-Einstellungen für ReFrames        : 2 frames
    Format_Settings_GOP                      : M=2, N=30
    Codec-ID                                 : avc1
    Codec-ID/Info                            : Advanced Video Coding
    Dauer                                    : 3min 49s
    Bitrate                                  : 3 068 Kbps
    Breite                                   : 640 Pixel
    Höhe                                     : 480 Pixel
    Bildseitenverhältnis                     : 4:3
    Modus der Bildwiederholungsrate          : konstant
    Bildwiederholungsrate                    : 29,970 (29970/1000) FPS
    ColorSpace                               : YUV
    ChromaSubsampling/String                 : 4:2:0
    BitDepth/String                          : 8 bits
    Scantyp                                  : progressiv
    Bits/(Pixel*Frame)                       : 0.333
    Stream-Größe                             : 84,1 MiB (89%)
    Sprache                                  : Englisch
    Kodierungs-Datum                         : UTC 2010-01-04 07:43:32
    Tagging-Datum                            : UTC 2010-01-04 08:22:19
    colour_range                             : Limited
    colour_primaries                         : BT.601 NTSC
    transfer_characteristics                 : BT.709
    matrix_coefficients                      : BT.601

    Audio
    ID                                       : 2
    Format                                   : ADPCM
    Format-Einstellungen für Firm            : IMA
    Codec-ID                                 : ima4
    Dauer                                    : 3min 49s
    Bitraten-Modus                           : konstant
    Bitrate                                  : 375 Kbps
    Kanäle                                   : 2 Kanäle
    Kanal-Positionen                         : Front: L R
    Samplingrate                             : 44,1 KHz
    BitDepth/String                          : 16 bits
    Stream-Größe                             : 10,3 MiB (11%)
    Sprache                                  : Englisch
    Kodierungs-Datum                         : UTC 2010-01-04 07:43:35
    Tagging-Datum                            : UTC 2010-01-04 08:22:19

    Andere
    ID                                       : 3
    Type                                     : Time code
    Format                                   : QuickTime TC
    Dauer                                    : 3min 49s
    TimeCode_FirstFrame                      : 00:00:00;00
    TimeCode_Striped/String                  : Ja
    Sprache                                  : Englisch
    Kodierungs-Datum                         : UTC 2010-01-04 08:22:19
    Tagging-Datum                            : UTC 2010-01-04 08:22:19

**Comment by mbunkus** [comment]
I will need access to that file if you want me to do anything about it. Please upload it to my [FTP server](/mbunkus/mkvtoolnix/wiki/FTP-server). Thanks.

**Comment by Klaus1189** [comment]
Ich hoffe, dass es so auch geht
https://drive.google.com/file/d/0B4y5u0gBKYqbRFFjemVkRzFnaVE/view?usp=sharing

**Comment by mbunkus** [comment]
Sure, thanks.

**Comment by mbunkus** [comment]
Hmm I don't see what's wrong with that file. mkvmerge can read and mux it just fine. What exactly did you do? What exactly didn't work the way you expected it to (ideally post screenshots of error messages)? Which version of MKVToolNix are you using?

**Comment by Klaus1189** [comment]
At first I tried to mux it via batch two times, it failed, but I don't know why, then I opened it in GUI and also an error occured, "keine Spuren zum Muxen gefunden". As you told me it is fine, I tried it again and I know what was wrong - it was me (30 cm vorm Bildschirm) while using the GUI no streams were selected by default and I missed that. 
It works now with the GUI and with the exact same batch file (one line), but I don't know what was the error during batch file usage the two times I tried it.
Anyway it is working and this issue can be closed. Thank you and I apologize for the issue :blush:
